packages/paste-style-props/src/types/flexbox.ts

```diff
@@ -95,19 +81,8 @@ export type Flex = ResponsiveValue<FlexOptions>;
  *
  * [MDN reference](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-self)
  */
-export type JustifySelfOptions =
-  | CSS.Globals
-  | CSS.SelfPosition
-  | 'auto'
-  | 'baseline'
-  | 'left'
-  | 'normal'
-  | 'right'
-  | 'stretch'
-  | string;
-export type JustifySelf =
-  | ResponsiveValue<CSS.Globals | CSS.SelfPosition | 'auto' | 'baseline' | 'left' | 'normal' | 'right' | 'stretch'>
-  | ResponsiveValue<string>;
+export type JustifySelfOptions = Property.JustifySelf;
```

Comment:
This one seems fine, we add `string` to the end so we might as well use the built-in property